lib/cc/service/response_check.rb

```diff
@@ -19,13 +19,13 @@ class ResponseCheck < Faraday::Response::Middleware
     def on_complete(env)
       if ErrorStatuses === env[:status]
         message = error_message(env) ||
-          "API request unsuccessful (#{env[:status]})"
+                  "API request unsuccessful (#{env[:status]})"
```

Comment:
This too. I can understand why it'd get flagged, but the fix of pushing it over further doesn't seem right.